run.py

```diff
@@ -365,6 +365,7 @@ def start_cluster(nodes: Sequence[Node]):
         logger.info('Letting stressor run for a while...')
         time.sleep(5)
 
+        start_datetime = (datetime.datetime.now().astimezone() - datetime.timedelta(minutes=5)).isoformat()
```

Comment:
Does the script do anything with this variable? I can't see it being used anywhere.